### PR TITLE
feat: #489 allow multiple rewrites in single annotation

### PR DESCRIPTION
**Pull request by dosmanak**

### CODE REVIEW

**Comment by ivanmatmati** [comment]
Can you also adapt the documentation part relative to this annotation ? It's located in documentation/doc.yaml. Then run `make doc`

File: pkg/annotations/ingress/reqPathRewrite.go [hunk 1]
```diff
@@ -27,23 +27,25 @@ func (a *ReqPathRewrite) Process(k store.K8s, annotations ...map[string]string)
 	if input == "" {
 		return
 	}
-	parts := strings.Fields(strings.TrimSpace(input))
+	for _, rule := range strings.Split(input, "\n") {
```
[the remark this answers is not in the document]
Comment (dosmanak) [reply pkg/annotations/ingress/reqPathRewrite.go:30]: there is TrimSpace call on the next line that will strip spaces before expolding into parts for rewrite rule creation. It is not sufficient?

**Comment by dosmanak** [comment]
> Can you also adapt the documentation part relative to this annotation ? It's located in documentation/doc.yaml. Then run `make doc`

Thank you for pointing this out. I pushed a new commit with docs.

**Comment by dosmanak** [comment]
I did not find test for annotation that I could enrich.

File: pkg/annotations/ingress/reqPathRewrite.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (ivanmatmati) [reply pkg/annotations/ingress/reqPathRewrite.go:30]: No, because you must avoid to loop over an empty rule otherwise you get the error message. Look at the log output, you should see it.

**Comment by ivanmatmati** [comment]
> I did not find test for annotation that I could enrich.

Yes, indeed there isn't one. If you want to tackle this part,you can try to create an integration test according the existing one in deploy/test/integration.

File: pkg/annotations/ingress/reqPathRewrite.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (dosmanak) [reply pkg/annotations/ingress/reqPathRewrite.go:30]: Thank you for clarification.
I pushed a fixup, can be squashed when ready.
I have not prepared any test environment for this PR :-( so I did not test it properly.
Or is it visible somewhere in github CI log?

**Comment by ivanmatmati** [comment]
Can you rephrase your commit message without "fixup !"? You can use "MINOR: " as the header of message.
